### ISSUE TITLE
History graph is flat across runs with different values

### ISSUE THREAD

**Issue by radeesh** [jenkinsci/nested-data-reporting-plugin#3]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
  Jenkins: 2.346.1
  OS: Linux - 4.14.281-212.502.amzn2.x86_64
  ---  
  nested-data-reporting:3.4.0
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

Linux - 4.14.281-212.502.amzn2.x86_64

### Reproduction steps

Ran multiple jobs with different values for lt5 and gt5

publishReport jsonString: '{"items":[{"id":"payment","name":"Payments","result":{"lt5":99,"gt5":1}},{"id":"shipping","name":"Shipping","result":{"lt5":99,"gt5":1}}],"colors":{"gt5":"#EF9A9A","lt5":"#FFF59D"}}'

### Expected Results

Trend graph inside the job should show value of each run, similar to chart on main trend graph in main page.
<img width="499" alt="expected" src="https://user-images.githubusercontent.com/4131979/180472193-8e9c0822-c5e5-4e4e-b365-53376a9f8199.png">

### Actual Results

History trend graph inside the job shows latest value for all runs (current and prior runs). The graph is flatlined across all runs same values.
<img width="733" alt="actual" src="https://user-images.githubusercontent.com/4131979/180472209-b86a7e35-c822-4028-9224-8562f02e6fc3.png">

**Comment by simonsymhoven** [comment]
Thanks for the hint! Fixed it with [PR-4](https://github.com/jenkinsci/nested-data-reporting-plugin/pull/4).

**Comment by simonsymhoven** [comment]
resolved by #4.
